### ISSUE TITLE
Mega Hard Light Field and Power-able Turret

### ISSUE THREAD

**Issue by IceTudor** [BEEmod/BEE2-items#2285]
Mega Hard Light Field: A fizzler, which, works exactly like a light bridge, but the size is that of a fizzler.
Power-able Turret: A turret which activates and deactivates using connections:
Start enabled, opposite of input, simple.
Start reversed, if connected to something else, when destroyed, it will activate/deactivate its reciever

**Comment by awesomedoctorx3** [comment]
1: Force Deflection Field.
2: planned on.

**Comment by IceTudor** [comment]
The Force Deflection Field doesn't block bullets. It's more of a grating.

**Comment by IceTudor** [comment]
The Force Deflection Field doesn't block bullets. It's more of a grating.

**Comment by Endermage77** [comment]
I suggested the Shutdown Turret (as I call it) earlier, sir.
See #1682 for that particular issue.
See also #1612 for a discussion about turrets that fizzle themselves upon getting "killed," like the turret dropper turrets do.
See also #1150 for a discussion about variants on the normal turret such as a sniper turret or a super-deadly turret.

**Comment by IceTudor** [comment]
@Endermage77 
For 1. That's in march! I would've had no idea of it, as I still had the old BEE2!
For 2. They don't fizzle themselves, they just activate/deactivate what they were sending to. Which wasn't in your suggestion, so it was more of a semi-duplicate.

**Comment by Endermage77** [comment]
Indeed, the turret you suggested does not fizzle itself. I was merely presenting you with further reading about related topics.

**Comment by IceTudor** [comment]
Hmm. Is it possible to make turrets with customize-able FoV and Range?

**Comment by Endermage77** [comment]
Range, absolutely. FoV, I'm not certain about.

The problem though, stems from the Duck Test.
That is, if a person sees any turret and it behaves vaguely like a turret, they'll assume that it's a perfectly ordinary turret.
And then they find out it isn't and get roaring mad at you for not making this more obvious and dying to the turret's wacky-weird functions.


Take the fizzlers, for instance. If they all looked like the normal fizzler, imagine how frustrated you'd get trying to tell which fizzler does exactly what. This fizzler doesn't fizzle cubes but fizzles portals, that one blocks instead of fizzles, this one only blocks people...
Imagine the insanity if all fizzler types looked exactly the same.

The same problem applies to turrets. Yes, you can add all sorts of skins and signage onto a turret, but from a distance most people only look at a turret once and see a normal turret without bothering to check that judgment. Or perhaps it's too dark to be able to tell.

I hope you understand what I'm trying to get across?

**Comment by ZombiePwner1** [comment]
@Endermage77 I'd have to agree with endermage there'd be quite abit of issues with this.

**Comment by IceTudor** [comment]
Maybe for FoV the eye could be wider, and for range the turret larger

**Comment by Endermage77** [comment]
Sure. Something that immediately gives away that this is not a normal turret at first glance from far away and/or in a dark environment

**Comment by Axolotlus** [comment]
I'll suggest that it should use a different model from the basic turret, one different enough to distinguish in any level of visibility. Maybe something spherical, akin to the rocket turret from P1, except with legs?

**Comment by awesomedoctorx3** [comment]
make it bige

**Comment by IceTudor** [comment]
Could it be possible to make the ceiling turrets scrapped from the game? Kinda like an upside down rocket turret but with bullets.

**Comment by IceTudor** [comment]
Or maybe a homing aerial faith plate, which launches things at you.

**Comment by Endermage77** [comment]
Like the one from the end of Twelve Angry Tests?

**Comment by IceTudor** [comment]
Yep. Got the idea from that.

**Comment by Endermage77** [comment]
May want to make a separate issue for that, then.
